client/src/components/Tip/Tip.scss

```diff
@@ -0,0 +1,20 @@
+.tip.title-tip {
+  padding: 0;
+  margin: -2px 0 0 5px;
+  width: 16px;
+
+  // this selector is used to get priority over .btn-secondary:not(:disabled)
+  &:not(:disabled) {
+    &:hover,
+    &:not(:hover),
+    &:active,
+    // click and hold
+    &:active:hover,
+    // click and hold and move mouse off element
+    &:active:not(:hover) {
+      background: transparent;
```

Comment:
You could probably avoid this by conditionally changing the button `color` prop to `link`.

Reply:
Changing the color prop on the `<Button>` component doesn't fix this